extensions/import/src/wizard/flatFileWizard.ts

```diff
@@ -99,24 +99,17 @@ export class FlatFileWizard {
 
 		this.importAnotherFileButton.hidden = true;
 		this.wizard.customButtons = [this.importAnotherFileButton];
-
 		this.wizard.onPageChanged(async (event) => {
-			let idx = event.newPage;
-
-			let page = pages.get(idx);
-
-			if (page) {
-				page.setupNavigationValidator();
-				page.onPageEnter();
+			let newPageIdx = event.newPage;
+			let lastPageIdx = event.lastPage;
+			let newPage = pages.get(newPageIdx);
+			let lastPage = pages.get(lastPageIdx);
+			if (lastPage) {
+				await lastPage.onPageLeave();
```

Comment:
>await lastPage.onPageLeave(); [](start = 4, length = 29)

Bleh - so there's a return value here but it's just a boolean that seems to indicate whether the page enter/leave/etc was successful. But what does that even mean? So in general I'd say we should check the return value and do something if it's false but I can't think a great way to handle this. At the very least consider logging it to console.